app/services/diagnostics/move_inspector.rb

```diff
@@ -338,6 +360,16 @@ def generate
       @output
     end
 
+    def diff(old_paper_trail, new_paper_trail)
+      old_values = old_paper_trail.attributes
```

Comment:
As I understand it reify is not needed here as this is a Paper Trail object.

Reply:
`reify` is still needed here, as it stands this will get the attributes of the Version object